src/core/generator.ts

```
@@ -913,10 +1069,26 @@ ${migrationNote}${declarationMethodsSection}
     ]);
     const functionResults = [...module.functions]
       .sort((a, b) => a.name.localeCompare(b.name))
```

ESLint

[error] Use module.functions.toSorted() instead of copying and sorting

(e18e/prefer-array-to-sorted)